app/utils/lib/isURL.js

```diff
@@ -1 +1 @@
-export const isURL = (str) => /^https?:\/\//.test(str);
+export const isURL = (str) => /^(https?|data):\/\//.test(str);
```

Comment:
This RegEx doesn't seem to be catching `data` URIs correctly 🤔 

https://regex101.com/r/93fBt3/2

Reply:
Ouch, I completely ignored that host prefix!